.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM rust:1.50
+
+# versions (according to their git tag) of different tools to download
+# https://github.com/EmbarkStudios/cargo-deny/releases
+ARG CARGO_DENY_VER=0.9.0
```

Comment:
We're currently using cargo deny 0.8.5 in the pr.yml. I just tested and 0.9.0 does work with our repo (I know we had some issues with another version), so we should probably make these match?

Reply:
Yes, I was sure to test it out first. I've now updated the `pr.yml` to match.